testing/src/main/java/io/stargate/it/http/docsapi/BaseDocumentApiV2Test.java

```diff
@@ -2073,42 +2397,6 @@ public void testInvalidFullDocPageSize() throws IOException {
             "{\"description\":\"The parameter `page-size` is limited to 20.\",\"code\":400}");
   }
 
-  @Test
-  public void createCollection() throws IOException {
```

Comment:
nit: could we re-arrange the code to avoid this showing up as a change? Actual code of `createCollection` appears to be the same :thinking:

Reply:
I moved it up in the order to separate the search stuff to be at the bottom of the class.. I grouped it in the nested test, but it did not work somehow.. Let's merge as it is..